config/clients/python/template/asyncio/rest.mustache

```diff
@@ -0,0 +1,237 @@
+{{>partial_header}}
+
+import io
+import json
+import logging
+import re
+import ssl
+
+import aiohttp
```

Comment:
This is not present in the deps file

Reply:
I don't think we use this file.  Let me gut it.